### PR TITLE
Export all building data

### PR DESCRIPTION
**Pull request by cdccollins**

### CODE REVIEW

File: spec/exporters/building_exporter_spec.rb [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+  let!(:accepted_survey) { FactoryBot.create(:survey, building: accepted_building, **MOCK_ACCEPTED_SURVEY_ATTRIBUTES) }
+  let!(:accepted_notification) do
+    FactoryBot.create(:notification, building: accepted_building, notification_mean: "letter", sent_at: "2022-01-18 16:15:07.433371 +0000")
+  end
+
+  path = File.expand_path("#{File.dirname(__FILE__)}/building_csv_mock.csv")
+
+  describe "mapping" do
+    before do
+      travel_to Time.zone.local(2022, 1, 21) do
```
Comment (allbecauseyoutoldmeso) [inline spec/exporters/building_exporter_spec.rb:155]: 🙌

**Review by allbecauseyoutoldmeso — APPROVED** [review]
looks good!

File: app/exporters/building_exporter.rb [hunk 2]
```diff
@@ -0,0 +1,105 @@
+# rubocop:disable all
```
[the remark this answers is not in the document]
Comment (cdccollins) [reply app/exporters/building_exporter.rb:1]: It gets a load of rubocop warnings about the file/methods being too long, so I just copied what was already in the delta_csv_exporter 😂

File: app/models/building.rb [hunk 3]
```diff
@@ -79,7 +80,7 @@ def address
       street,
       city_town,
       postcode
-    ].reject(&:blank?).join("\n")
+    ].compact_blank.join("\n")
```
[the remark this answers is not in the document]
Comment (cdccollins) [reply app/models/building.rb:83]: Neither did I, rubocop told me 😂

File: app/exporters/building_exporter.rb [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    city_town
+    postcode
+    land_registry_title_number
+    land_registry_proprietor_name
+    land_registry_proprietor_address
+    land_registry_proprietor_category
+    land_registry_proprietor_company_registration_number
+  ]
+
+  NOTIFICATION_FIELDS = (1..10).flat_map do |i|
```
[the remark this answers is not in the document]
Comment (cdccollins) [reply app/exporters/building_exporter.rb:22]: Yep good point, will increase

**Pull request merged**